### DISCUSSION TITLE
Not able to import figma file to unity

### DISCUSSION THREAD

**Discussion by PiyushGamer14** [simonoliver/UnityFigmaBridge#23] [Q&A] [answered]
Not able to import the figma file to unity ,tried every step in this video : https://www.youtube.com/watch?v=4LjvsMXwaI8
but still it getting stuck at importing and error comes in unity console that null reference related to google fonts ..please help me with this ,anyone facing this issue ??

**Comment by simonoliver** [comment]
I'd be happy to take a look - can you share a link to the figma document?

**Reply by PiyushGamer14** [reply]
hello when i import any document whether it is small or large still it shows null reference ...the document i used was in the video mentioned above 
![FigmaError](https://github.com/simonoliver/UnityFigmaBridge/assets/137914579/5dbad070-2964-4054-8ab2-79b0737144cb)

please help with the issue as everytime i try it alwys shows this message even i imported the fonts folder also

**Reply by simonoliver** [reply]
Can you share the link to the Figma file with me? That would help me find the problem and fix the issue. If it needs to be private, you can send me a DM on Twitter (simonoliveruk) or Discord (simono)

**Reply by PiyushGamer14** [reply]
yes ,

**Reply by PiyushGamer14** [reply]
yes sure

**Reply by simonoliver** [reply]
Hi - for some reason it's not able to load the font JSON in the Unity package here. "Packages/com.simonoliver.unityfigma/UnityFigmaBridge/Assets/google-fonts.json" which should work just fine. A few questions:

* Have you moved or renamed the package?
* Did you add the package as a GIT repository or add from disk?
* What version of Unity are you using?
* What platform? (Mac/PC/Linux)
* Can you try the example project here? https://github.com/simonoliver/UnityFigmaBridgeExample and see if it works?

Also you might want to delete the image in your post above as it shares your Figma API key

**Reply by PiyushGamer14** [reply]
no sir, i just downladed the package from github and imported to unity 
im using unity 2021.3.1
pc platform

so that stucking in import is due to json not being loaded ??

**Reply by simonoliver** [reply, marked as answer]
Ah this might be the problem - did you just add the files from the zip to your project folder? You need to add the package using the instructions here (rather than adding the downloaded files). Try deleting the files in your assets folder and follow the instructions here - 
 https://github.com/simonoliver/UnityFigmaBridge#setting-up

**Reply by PiyushGamer14** [reply]
now working fine had to install git and set environment variable and then had to just import the url and everything was working
just need to knw that transitions have not come it's only fade out ..will transitions also come or there is another method for the transition?

**Reply by simonoliver** [reply]
Great, glad its working - can you click "Mark as Answer" in the reply above? 
Right now the FIGMA API doesnt have the data we need to implement other transitions (such as transition type) but I've requested that they add it, so hopefully we'll be able to add in the future.

**Reply by PiyushGamer14** [reply]
yes sir installed it from the package manager now all working fine ..but just the transitions i made in figma are not imported to unity ..is there any method to import the exact figma transitions to unity

**Reply by simonoliver** [reply]
Right now the information to create those transitions is not available via the Figma API, so you have to create them manually

**Reply by PiyushGamer14** [reply]
ok ok thanks alot for the help .. 💯

**Reply by PiyushGamer14** [reply]
yes clciked it.. ohh then i guess have to wait for them to implement the API ..but thanks alot for the help
